robot picks up plate from sink
robot moved from sink to countertop
robot puts down plate to countertop
plate already cleaned!
robot moved from countertop to sink
robot picks up bowl from sink
robot moved from sink to countertop
robot puts down bowl to countertop
bowl already cleaned!
robot picks up plate from countertop
robot moved from countertop to sink
robot puts down plate to sink
robot moved from sink to countertop
robot picks up bowl from countertop
robot moved from countertop to sink
robot puts down bowl to sink
Traceback (most recent call last):
  File "/home/shivam/Desktop/side_quests/generalisation_project/OKG/frameworks/GPT3.5/other_domains/for_knowledge_expansion/./LLM_only/final_run.py", line 9, in <module>
    pour_contents("plate", "sink")
  File "/home/shivam/Desktop/side_quests/generalisation_project/OKG/frameworks/GPT3.5/other_domains/for_knowledge_expansion/LLM_only/functions/actions.py", line 2229, in pour_contents
    raise RuntimeError(f"pour_contents({from_rec}, {to_rec}) #{to_rec} is not a valid receptacle for this action!")
RuntimeError: pour_contents(plate, sink) #sink is not a valid receptacle for this action!
